ledfx/devices/utils/rgbw_conversion.py

```diff
@@ -0,0 +1,60 @@
+from enum import Enum
+
+import numpy as np
+
+
+class OutputMode(str, Enum):
+    RGB = "RGB"  # RGB data
+    RGBW_None = "RGBW No White"  # No white channel, just set to 0
+    RGBW_ACCURATE = "RGBW Accurate"  # Color accuracy approach (minimum RGB)
+    RGBW_BRIGHTER = (
```

Comment:
I find use of brackets here jarring, its programmatically redundant, but assuming its there for layout formatting, was this the autoformatter that did this?

Reply:
Was done by the autoformatter